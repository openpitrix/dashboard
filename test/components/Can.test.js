import Can from 'components/Can';

describe('Can', () => {
  it('basic render', () => {
    const wrapper = render(
      <Can>
        <div>show content</div>
      </Can>
    );

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
