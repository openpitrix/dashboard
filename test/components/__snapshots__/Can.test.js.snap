// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Can basic render 1`] = `
<div>
  show content
</div>
`;
